scripts/osrd-compose.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env sh
```

Comment:
Same here

Reply:
Done in https://github.com/osrd-project/osrd/compare/3dbf7d69f53eab1915cd691738cb64fb70bc1edd..87aca65215c352c49f677f6a664beebbde1564b1